recipes/sphinxcontrib-versioning/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sphinxcontrib-versioning" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a5fe9b4e36020488d0d037fccc0b21aaf71b80425cad42ef4a5e5c3c193d3cd
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - sphinx-versioning = sphinxcontrib.versioning.__main__:cli
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - click
+    - colorclass
+    - sphinx
+
+test:
+  commands:
+    - sphinx-versioning --help
+
+about:
+  home: https://github.com/Robpol86/sphinxcontrib-versioning
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Please ask upstream to add a `MANIFEST.in` with the license file included so that it will be present in future `sdist`s.